scripts/export-dos-labs.py

```diff
@@ -11,25 +11,29 @@
 sys.path.insert(0, '.')
 
 from docopt import docopt
+from dmapiclient import DataAPIClient
+
 from dmscripts.helpers.env_helpers import get_api_endpoint_from_stage
 from dmscripts.helpers.framework_helpers import find_suppliers_with_details_and_draft_services
-from dmapiclient import DataAPIClient
+from dmscripts.export_dos_labs import append_contact_information_to_services
 
 if sys.version_info[0] < 3:
     import unicodecsv as csv
 else:
     import csv
 
+REQUIRED_CONTACT_DETAILS_KEYS = ['email', 'contactName', 'phoneNumber']
+
 
 def find_all_labs(client):
     records = find_suppliers_with_details_and_draft_services(client,
                                                              FRAMEWORK_SLUG,
                                                              lot="user-research-studios",
                                                              statuses="submitted"
                                                              )
+    records = append_contact_information_to_services(records, REQUIRED_CONTACT_DETAILS_KEYS)
```

Comment:
Might be marginally more efficient to append contact information after filtering by onFramework?

Reply:
Will change the order (no idea if it actually would be more efficient but no harm)